dashboards/proposal.md

```diff
@@ -0,0 +1,74 @@
+
+### Functionality
+
+As outlined in this [blog post](http://blog.ibmjstart.net/2015/08/22/dynamic-dashboards-from-jupyter-notebooks/), the intent of this proposal is to seed a repository with a proof of concept extension providing the following features:
+
+* **Dashboard Layout** where the notebook author can drag/drop and resize notebook cells in a grid layout to dynamically create the web application within Jupyter.
+* **Dashboard Transformation** where the notebook author can convert the notebook document into a web app that executes the same code as the origina notebook and retains the desired application layout. Packaging of the web app assets includes:
+	* Static web resources necessary for operation of the dashboard frontend
+	* Configuration for directing the frontend to an appropriate kernel execution environment (e.g., [tmpnb](https://github.com/jupyter/tmpnb), [kernel gateway](https://github.com/jupyter-incubator/proposals/pull/3) )
+	* Assets to ease deployment of the dashboard frontend using common cloud technologies (e.g., Dockerfile, Cloud Foundry manifest)
+
+After publishing the additional prototype, improvements will occur within the incubator project in order to mature the features for later integration into Jupyter Notebook or promotion as other mainline components.
+
+## Audience
+
+* Notebook users who wish to switch to a dashboard view for easier interaction with widgets in their notebooks (e.g., during data exploration)
+* Notebook users who wish to layout and present their results in a dashboard view
+* Notebook users who wish to share interactive dashboard views with other notebook users when passing around notebook documents
+* Non-notebook users who want to interact with findings from analyses done in notebooks without running Jupyter Notebook themselves
+* New Notebook users who wish to use it as a light-weight application composition environment
+
+## Other options
+
+* Any number of commercial and open source dashboarding applications exist (e.g., Tableau). Some have the ability to produce dashboards specifically from notebooks (e.g., Mathematica Cloud, Apache Zeppelin, Databricks).
+* The traditional approach of doing analysis in notebooks and then stepping outside to create a traditional web application is always an option.
+* One could use [thebe](https://github.com/oreillymedia/thebe) or an equivalent JS library to manually create a web application that uses Jupyter kernels for code execution. This project automates this otherwise manual effort. 
+
+## Integration with Project Jupyter
+In accordance with the [New Subproject Process](https://github.com/jupyter/governance/blob/master/newsubprojects.md#incubation-of-subprojects), this incubation proposal pertains to an idea that has yet to be vetted with the community. 
```

Comment:
I think another area of integration will be with ipywidgets and the display system (for rich output), which today provides most of the functionality/UI in those dashboards.